README.md

```diff
@@ -136,6 +124,14 @@ See the following command to use for this specific codebase:
 
 docker run --env-file ./.env -p 5000:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:dev
 
+Run all tests inside Docker:
+
+Build the Container:
+docker build --tag todo-app:devtest --target devtestpy .
+
+Run the tests in Docker:
+docker run --env-file ./.env -p 5000:5000 todo-app:devtest
```

Comment:
Thanks for adding these clear instructions, although I think we could improve the run command further:
* Do we need to expose a port? Do we actually connect to the app from outside the container at all?
* Is the `.env` file the right file to pass through? We don't have any tests that should legitimately talk to Trello, so while the mocking _should_ prevent requests firing, it would be safer to pass through fake credentials so anything we miss fails. Using your `.env.test` file instead might make more sense here